src/commands/android/subcommands/install/index.ts

```diff
@@ -1,8 +1,22 @@
+import inquirer from 'inquirer';
+
+import Logger from '../../../../logger';
 import {Options, Platform} from '../../interfaces';
+import {verifyOptions} from '../common';
 import {installApp} from './app';
 import {createAvd} from './avd';
 
 export async function install(options: Options, sdkRoot: string, platform: Platform): Promise<boolean> {
+  const optionsVerified = verifyOptions('install', options);
+  if (!optionsVerified) {
+    return false;
+  }
+
+  const subcommandFlag = optionsVerified.subcommandFlag;
+  if (subcommandFlag === '') {
+    await flagsPrompt(options);
+  }
```

Comment:
Since we are getting `subcommandFlag` from `optionsVerified`, we should rely on the value it contains and not on the `options` properties in the code that follows.